app/selfserve/module/Olcs/src/Controller/Lva/AbstractUploadEvidenceController.php

```diff
@@ -358,4 +359,31 @@ private function shouldShowSupportingEvidence()
         }
         return $this->application['status']['id'] === RefData::APPLICATION_STATUS_UNDER_CONSIDERATION;
     }
+
+    /**
+     * Has any evidence been uploaded
+     *
+     * @param array $data
+     * @return bool
+     */
+    function hasEvidence($data)
```

Comment:
small improvement suggestion (php type hints instead of docblocks)

```suggestion
     */
    function hasEvidence(array $data): bool
```